### PR TITLE
Add the scripts used for lightning docs

### PR DESCRIPTION
**Pull request by qihqi**

### CODE REVIEW

**Review by ManfeiBai — APPROVED** [review]
Thanks, LGTM

File: experimental/torch_xla2/docs/understand_jax_jit/jax_jit.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
```
Comment (ManfeiBai) [inline experimental/torch_xla2/docs/understand_jax_jit/jax_jit.py:1]: do we want to keep all the comments too?

File: experimental/torch_xla2/docs/understand_jax_jit/torch_module.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import jax
```
Comment (ManfeiBai) [inline experimental/torch_xla2/docs/understand_jax_jit/torch_module.py:1]: do we want to keep all the comments too?

File: experimental/torch_xla2/docs/understand_jax_jit/jax_jit.py (hunk 1 above)
Comment (qihqi) [reply experimental/torch_xla2/docs/understand_jax_jit/jax_jit.py:1]: yes

**Pull request merged**